packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -170,18 +160,20 @@ const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown
     const deleteToken = async (token: string) => {
         setState({ is_delete_loading: true });
 
+        clearTimeout(timeout_deletetokens);
+
         const token_response = await WS.authorized.apiToken({ api_token: 1, delete_token: token });
 
         populateTokenResponse(token_response);
 
         if (isMounted()) setState({ is_delete_loading: false, is_delete_success: true });
 
-        setTimeout(() => {
+        timeout_deletetokens = setTimeout(() => {
```

Comment:
```suggestion
        timeout_ref.current = setTimeout(() => {
```

Reply:
Now we can clear the timeout in a cleanup function from a useEffect
```suggestion
React.useEffect(() => {
    return () =>  clearTimeout(timeout_ref.current)
})